workers/unity/Assets/Playground/Scripts/Worker/SteamClientWorkerConnector.cs

```diff
@@ -0,0 +1,181 @@
+using System;
+using Improbable.Gdk.Core;
+using Improbable.Worker;
+using UnityEngine;
+
+namespace Playground
+{
+    /// <summary>
+    ///     The SteamClientWorkerConnector class provides a sample solution for connecting a game client published
+    ///     on Steam to a SpatialOS deployment. It provides example code for retrieving a
+    ///     Steam auth session token at runtime using either Steamworks.NET (https://steamworks.github.io) or
+    ///     Facepunch.Steamworks (https://github.com/Facepunch/Facepunch.Steamworks).
+    ///     You can also use the ClientWorkerConnector parent class to parse the Steam auth session token
+    ///     from the commandline arguments in case the Steam auth session token is known in advance.
+    ///     You will need to install either Steamworks.NET (https://steamworks.github.io) or
+    ///     Facepunch.Steamworks (https://github.com/Facepunch/Facepunch.Steamworks) in order to use this code.
+    /// </summary>
+    public class SteamClientWorkerConnector : ClientWorkerConnector
```

Comment:
should this inherit from `ClientWorkerConnector`? wouldn't it be better to inherit directly from `WorkerConnector`?